.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -7,7 +7,9 @@ assignees: ''
 
 ---
 
-<!-- 🚨 Please provide detailed information and Do Not skip any instructions as they are all required and essential to help us understand the feature 🚨 -->
+<!-- 🚨 PLEASE PROVIDE DETAILED INFORMATION AND DO NOT SKIP ANY INSTRUCTIONS AND INFORMATION MENTIONED BELOW AS THEY ARE ALL REQUIRED AND ESSENTIAL TO HELP US UNDERSTAND THE FEATURE.
+...
+IF NOT CERTAIN ABOUT THE FEATURE AND REQUIRE MORE CLARITY THEN PLEASE POST ON "IDEAS" CATEGORY OF THE DISCUSSIONS PLATFORM [https://github.com/windows-toolkit/WindowsCommunityToolkit/discussions/categories/ideas] WHERE YOU CAN DISCUSS AND ENAGAGE WITH THE COMMUNITY TO GAIN FURTHER CLAIRITY REGARDING THE FEATURE 🚨 -->
```

Comment:
Great idea @Kyaa-dost!

Thoughts on if we should be more explicit about starting feature requests for things like new controls explicitly in the discussions area? As we had mentioned about trying to eventually have issues just be things we know community folks or the toolkit team are working on?

Reply:
@michael-hawker so you mean any in-progress or already developed feature by the community member be used as a feature request in the issues? Whereas any idea/s about a feature be opened in discussions?